src/main/java/uk/gov/hmcts/dts/fact/model/AdditionalLink.java

```diff
@@ -10,9 +12,17 @@
 public class AdditionalLink {
     private String url;
     private String description;
+    private String location;
 
     public AdditionalLink(uk.gov.hmcts.dts.fact.entity.AdditionalLink additionalLink) {
         this.url = additionalLink.getUrl();
         this.description = chooseString(additionalLink.getDescriptionCy(), additionalLink.getDescription());
+
+        final SidebarLocation location = additionalLink.getLocation();
+        if (location != null && StringUtils.isNotBlank(location.getName())) {
```

Comment:
Can we have a null sidebar location? I imagine it'll either go in one or two?

Reply:
I make the sidebarLocation column on the database table NOT NULL so it will never be null. I guess this is a redundant check here